ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/NetworkProtocolVersion.hs

```diff
@@ -47,6 +50,8 @@ instance SupportedNetworkProtocolVersion (ShelleyBlock proto era) where
       , (NodeToClientV_10, ShelleyNodeToClientVersion5)
       , (NodeToClientV_11, ShelleyNodeToClientVersion5)
       , (NodeToClientV_12, ShelleyNodeToClientVersion5)
+      , (NodeToClientV_13, ShelleyNodeToClientVersion5)
+      , (NodeToClientV_14, ShelleyNodeToClientVersion6)
```

Comment:
`NodeToClientV_13` was missing, so I added it.  Could I have done this in error?

Reply:
The list for `ShelleyBlock` probably doesn't matter too much -- it's only ever used for `testnet`s.

Enabling the `LocalTxMonitor` and your new queries probably makes sense though 👍